docs/source/user-guide/vision/vision_properties.rst

```diff
@@ -32,6 +32,14 @@ Example for specific scenarios in which measuring properties may come in handy:
    the snow, the brightness of the image may be used to predict the label "wolf" easily. In this case, a model
    might not learn to discern wolf from dog by the animal's characteristics, but by using the background color.
 
+Passing Properties to the Checks
+================================
+The properties can be passed to the check in one of two ways:
+
+#. **Properties** - a list of functions to be calculated on the data during the check (specific format listed below)
+   passed to the check init.
+#. **Static Properties** - a dictionary with the result of pre calculated properties per sample passed to the check run.
```

Comment:
Should be named "Pre-Calculated Properties" IMO
Maybe a word here on why I'd want that, or alternatively (as this is explained in the relevant section) link to the section, and just way "Properties calculated per sample before running the check. Read more ..."